types/imagemin-svgo/index.d.ts

```diff
@@ -1,12 +1,16 @@
-// Type definitions for imagemin-svgo 7.0
+// Type definitions for imagemin-svgo 8.0
 // Project: https://github.com/imagemin/imagemin-svgo#readme
 // Definitions by: Romain Faust <https://github.com/romain-faust>
+//                 Piotr Błażejewicz <https://github.com/peterblazejewicz>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
-// TypeScript Version: 2.2
 
 import { Plugin } from 'imagemin';
 import { Options as SvgoOptions } from 'svgo';
 
+/**
+ * SVGO imagemin plugin
+ * @async
```

Comment:
For my own edification, what does `@async` mean here?  It doesn't look promise-returning.

Reply:
@amcasey so it shoudl denote declaration done with `async` keyword, but I was wrong and taken outcome details, this should not be here (in this specific case), thx for spotting this!
```suggestion
```